src/dymoprint/lib/unicode_blocks.py

```diff
@@ -5,25 +5,35 @@
 LH = "▄"
 FB = "█"
 NB = "\xa0"
+LS = "░"
 
 assert UH == "\N{UPPER HALF BLOCK}"
 assert LH == "\N{LOWER HALF BLOCK}"
 assert FB == "\N{FULL BLOCK}"
 assert NB == "\N{NO-BREAK SPACE}"
+assert LS == "\N{LIGHT SHADE}"
 
+BLACK_PIXEL = (255, 255, 255, 255)
+WHITE_PIXEL = (0, 0, 0, 255)
+# when the image height is odd, the preview image height is extended by 1 pixel
+# this padding result in the added line being composed of transparent pixels.
+TRANSPARENT_PIXEL = (0, 0, 0, 0)
 
 dict_unicode = {
-    (0, 0): FB,
-    (1, 0): LH,
-    (0, 1): UH,
-    (1, 1): NB,
+    (BLACK_PIXEL, BLACK_PIXEL): FB,
+    (WHITE_PIXEL, BLACK_PIXEL): LH,
+    (BLACK_PIXEL, WHITE_PIXEL): UH,
+    (WHITE_PIXEL, WHITE_PIXEL): NB,
+    (BLACK_PIXEL, TRANSPARENT_PIXEL): UH,
+    (TRANSPARENT_PIXEL, TRANSPARENT_PIXEL): NB,
```

Comment:
I believe only the second pixel can be transparent, right?
```suggestion
    (WHITE_PIXEL, TRANSPARENT_PIXEL): NB,
```

We should probably be testing this with an odd-length payload that actually touches the edge.

Reply:
`dymoprint --preview-inverted "single line"` should cover it (in current setup it results in an off image)